lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,100 @@
+require 'rake'
+
+$organization_id = 13856
+
+def remove_teem_group_member(token, tm_diff_users, all_user_ids, group_id)
+  tm_diff_users.each do |email|
```

Comment:
I would rename tm_diff_users. The fact that it's from a 'diff' isn't relevant to remove_teem_group_member().